src/Engine/Renderer/RenderObject/Primitives/DrawPrimitives.cpp

```diff
@@ -464,7 +464,7 @@ namespace Ra {
                 std::vector<uint> indices;
                 indices.reserve(pointCount * 2 - 2);
 
-                Scalar dt = 1.0 / Scalar(pointCount - 1);
+                Scalar dt = 1.0f / Scalar(pointCount - 1);
```

Comment:
I believe the correct way to do it is `Scalar(1) / / Scalar(pointCount - 1)`.
With your version, you implicitly assume that Scalar is float.